app/fissile.go

```diff
@@ -1101,3 +1103,66 @@ func (f *Fissile) generateKubeRoles(settings kube.ExportSettings) error {
 
 	return nil
 }
+
+// GraphBegin will start logging hash information to the given file
+func (f *Fissile) GraphBegin(outputPath string) error {
+	file, err := os.Create(outputPath)
+	if err != nil {
+		return err
+	}
+	_, err = file.WriteString("strict digraph {\n")
+	if err != nil {
+		return err
+	}
+	_, err = file.WriteString("graph[K=5]\n")
+	if err != nil {
+		return err
+	}
+	f.graphFile = file
+	return nil
+}
+
+// GraphEnd will stop logging hash information
+func (f *Fissile) GraphEnd() error {
+	if f.graphFile == nil {
+		return nil
+	}
+	if _, err := f.graphFile.WriteString("}\n"); err != nil {
+		return err
+	}
+	if err := f.graphFile.Close(); err != nil {
+		return err
+	}
+	f.graphFile = nil
+	return nil
+}
+
+// GraphNode adds a node to the hash debugging graph
```

Comment:
Should possibly note that this makes the `fissile` struct an implementation of the `ModelGrapher` interface.

Reply:
Comment added in e1f5e2f